Index.HTML

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="css/style.css">
+    <link href="https://fonts.googleapis.com/css?family=Roboto&display=swap" rel="stylesheet">
+    <title>YouTube</title>
+</head>
+<body>
+    <nav class="nav-bar">
+        <div id="logo-div">
+            <img src="https://archive.fo/Bss88/51a3832e6af6792add876216da444db23a4bea96.png" id="logo" alt="YouTube Logo">
+        </div>
+        <form>
+            <div id="search-div">
+                <input type="text" id="search-bar" placeholder="  Search">
+            </div>
+            <button type="submit" id="search-button">
```

Comment:
- **`[OPTIONAL]`** For simple components like search field, we can ignore the `<form>` tag. [more](https://stackoverflow.com/questions/3294572/is-input-well-formed-without-a-form)